Cesium3DTilesSelection/src/Tile.cpp

```diff
@@ -542,9 +557,19 @@ void Tile::processLoadedContent() {
       }
       // else if it has a model, try to get the copyright
       else if (this->_pContent->model->asset.copyright) {
-        this->_pContent->credit = externals.pCreditSystem->createCredit(
-            *this->_pContent->model->asset.copyright,
-            this->getTileset()->getOptions().showCreditsOnScreen);
+        if (!perTileCredits.has_value()) {
+          perTileCredits = std::make_optional<std::vector<Credit>>();
+        }
+        const auto& copyright = *this->_pContent->model->asset.copyright;
+        size_t start = 0;
+        size_t end;
+        do {
+          end = copyright.find(',', start);
```

Comment:
This should change from `,` to `;`. See https://github.com/CesiumGS/cesium/pull/10147